thunder/tests/opinfos.py

```diff
@@ -2548,7 +2548,7 @@ def div_sample_generator(op, device, dtype, requires_grad, **kwargs):
             pytest.mark.xfail,
             "test_core_vs_torch_consistency",
             executors=("nvfuser",),
-            dtypes=(datatypes.bool8, datatypes.bfloat16, datatypes.float16, datatypes.float32),
+            dtypes=(datatypes.bool8,),
```

Comment:
I'm not sure that we can delete the float16 case from this directive.  It's a bit flaky, failing 3/50 times with this change on my machine.   There's already [#1724](https://github.com/Lightning-AI/lightning-thunder/issues/1724) to track this. 

Would you mind also removing the TODO comment above?